<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RetChartWindow</class>
 <widget class="QWidget" name="RetChartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1305</width>
    <height>905</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Returns Chart</string>
  </property>
  <widget class="QPushButton" name="btn_back2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>870</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
